src/test/unit_tests/test_namespace_fs.js

```diff
@@ -472,6 +474,22 @@ mocha.describe('namespace_fs', function() {
 
         });
 
+        mocha.it('delete dir object without last slash - /x/y/z/', async function() {
+            const source = buffer_utils.buffer_to_read_stream(data);
+            await upload_object(ns_tmp, upload_bkt, upload_key_4, dummy_object_sdk, source);
+            await delete_object(ns_tmp, upload_bkt, '/x/y/z', dummy_object_sdk);
+
+            let entries;
+            try {
+                entries = await nb_native().fs.readdir(DEFAULT_FS_CONFIG, ns_tmp_bucket_path + dir_2);
+            } catch (e) {
+                assert.ifError(e);
+            }
+            console.log('stop when not empty - entries', entries);
```

Comment:
Do we need this printing in the CI?
I'm not sure I understand it, could you please a comment with an explnantion?